distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Regex/Pattern.enso

```diff
@@ -288,13 +288,11 @@ type Pattern
                 groups = self.internal_regex_object.groups
 
                 n = case groups of
-                        # If Nothing, there are no named groups
-                        Nothing -> Error.throw (No_Such_Group.Error name)
-                        _ ->
-                            qq = (read_group_map groups name)
-                            case qq of
-                                Nothing -> Nothing
-                                n : Integer -> n
+                    # If Nothing, there are no named groups
+                    Nothing -> Error.throw (No_Such_Group.Error name)
+                    _ ->
+                        n = read_group_map groups name
+                        if n.is_nothing then Nothing else n
```

Comment:
```suggestion
                        read_group_map groups name
```

I don't understand the `if n.is_nothing then Nothing else n`? Isn't it basically equivalent to `n` itself?

Reply:
Oops, result of a previous simplification.